types/tests/server-test.ts

```diff
@@ -22,6 +22,19 @@ export default function config(this: Server): void {
   this.options("/foo"); // $ExpectType void
   this.del("/foo"); // $ExpectType void
 
+  this.get('/foo', () => ({}))
+  this.get('/foo', () => 0)
+  this.get('/foo', () => 'foo')
+  this.get('/foo', () => true)
+  this.get('/foo', () => null)
+  this.get('/foo', () => ['foo'])
+  this.get('/foo', () => ['foo', 0])
+  this.get<number>('/foo', () => 0)
+  this.get<[number, string]>('/foo', () => [0, 'foo'])
+  
+  this.get<number>('/foo', () => false) // $ExpectError
+  this.get<[number, string]>('/foo', () => ['foo', 0]) // $ExpectError
+
```

Comment:
I hate to ask this, but would you mind running `yarn prettier:update`?  It looks like the `prettier:check` script does not check `.ts` files in CI, which I'll fix separately.

Reply:
I ran that which made quite a few changes. Please make sure they are correct.

Reply:
Yes I think it's right, but could you back out the changes to `types/tests/collection-test.ts`?  It's making a conflict because I already formatted that file in a separate PR.  Rebasing on latest master might help.  Thanks again.

Reply:
Reverted the changes to `collection-tests.ts` 👍